integration-tests/shard-operator-integration-tests/shard-operator-integration-tests-v2/src/test/resources/features/processor-resource.feature

```diff
@@ -0,0 +1,26 @@
+Feature: ManagedProcessor tests
+
+  Background:
+    Given create Namespace
+
+  @VIP
+  Scenario: ManagedProcessor resource
+    When deploy ProcessorResource:
+    """
+      apiVersion:  com.redhat.service.smartevents/v2alpha1
+      kind: ManagedProcessor
+      metadata:
+        name: "processor-name"
+      spec:
+        name: "processor-name"
+        flows:
+          - from:
+              uri: "rhose:bridge"
+              steps:
+              - to:
+                  uri: "sink-name"
+    """
+    Then the ProcessorResource "processor-name" exists within 3 minute
+    And the Deployment "processor-name" is ready within 3 minute
+    And the Service "processor-name" exists within 3 minute
+    And the BridgeExecutor "processor-name" is in condition "Ready" within 3 minutes
```

Comment:
Hi @lucamolteni is possible to expect ManagedProcessor as Deployment, Service and with status Ready? Same as  BridgeExecutor in v1.

Reply:
No, (I've written the same thing before)
ManagedProcessor is ready
Deployment for Camelntegration is ready
KNative Trigger is ready
KNative Service is ready

`from:
               uri: "rhose:bridge"` is not valid